tests/evaluate/test_validation.py

```diff
@@ -854,6 +856,36 @@ def test_validation_model_comparison_relative_threshold_should_pass(
             )
 
 
+def test_validation_thresholds_validation_thresholds_no_mock():
```

Comment:
this test fails on master because `mlflow.evaluate` compares the same model.

Reply:
We have some tests with `validation_thresholds` and `baseline_model`, but they mock `evaluate`.